src/patterns/question-pages/index.md.njk

```diff
@@ -83,6 +83,12 @@ For example, ‘About you’
 
 You can also learn more about how starting with [one thing per page](https://www.gov.uk/service-manual/design/form-structure#start-with-one-thing-per-page) helps users in the GOV.UK Service Manual.
 
+##### Asking 'filter' questions to help users answer a complex question
+
+A series of simple questions can be easier to answer than one complex question. Especially if parts of it aren’t relevant to all users. A 'filter' question will let the user skip parts of a question that do not apply to them so they don't get confused.
+
+For example, you could ask a user if they've travelled outside the UK before asking which countries they've travelled to. This would help a user skip looking at a long list of countries when they do not need to.
```

Comment:
Okay, slight revision @CharlotteDowns. I didn't realise we reached the bottom of our heading numbers!
```suggestion
#### Asking filter questions simplify a complex question

A series of simple questions can be easier to answer than one complex question. Especially if parts of it aren’t relevant to all users. A 'filter' question will let the user skip parts of a question that do not apply to them so they do not get confused.

For example, you could ask the user if they've travelled outside the UK before you ask them which countries they've travelled to. This would let the user skip looking at a list of countries when they do not need to.
```